apps/nextra/pages/en/index.mdx

```diff
@@ -4,6 +4,34 @@ searchable: false
 ---
 
 import { Landing } from "@components/landing";
+import {
+  coinsCodeSnippet,
+  fungibleAssetsCodeSnippet,
+  objectsCodeSnippet
+} from "@components/landing/codeSnippets";
+import { codeToHtml } from "shiki";
 
-<Landing />
+export const getStaticProps = async () => {
+  const highlight = (codeSnippet) => {
+    return codeToHtml(codeSnippet, {
+      lang: "move",
+      themes: {
+        light: "github-light",
+        dark: "github-dark",
+      },
+      defaultColor: false,
+    })
+  }
```

Comment:
Can we separate this out as a util so it can be reused?

Reply:
I broke it out into a util function